### ISSUE TITLE
Thoughts on a `logout_redirect` feature?

### ISSUE THREAD

**Issue by aaronhmiller** [Kong/kong-plugin-session#25]
I've noticed when using the logout method (in my case, added GET and called :8000/session-route?session_logout) an empty screen just sits there. Seems like upon logout, having an option to redirect the user elsewhere would be in order?

**Comment by bungle** [comment]
Sounds reasonable!

**Comment by darrenjennings** [comment]
@aaronhmiller agreed, we originally used this plugin to manually invoke via javascript XHR request.
